bigquery_etl/format_sql/tokenizer.py

```diff
@@ -872,13 +876,13 @@ class FieldAccessOperator(Operator):
     JinjaBlockMiddle,
     JinjaBlockEnd,
     JinjaStatement,
-    MaybeCaseSubclause,
-    CaseSubclause,
     BlockMiddleKeyword,
     BlockStartKeyword,
     BlockEndKeyword,
     AliasSeparator,
     TopLevelKeyword,
+    MaybeCaseSubclause,
```

Comment:
Is this change just changing the ordering?

Reply:
It's so the new `TopLevelKeyword` tokens for merge statement matched clauses work, otherwise `CaseSubClause` would incorrectly get applied to the `WHEN` in `WHEN MATCHED` / `WHEN NOT MATCHED`.